cdap-common/src/main/java/org/apache/twill/filesystem/FileContextLocation.java

```diff
@@ -80,6 +80,32 @@ public boolean createNew() throws IOException {
   }
 
   @Override
+  public boolean createNew(String permission) throws IOException {
```

Comment:
I checked the JIRA. CDAP-5328, the other class that is used is FileContextLocationFactory but that class has changes in CDAP that is not present in Twill. How to find out what classes to remove in CDAP?